mapknowledge/__init__.py

```diff
@@ -163,7 +163,11 @@ def __init__(self, store_directory=None,
             cache_msg = f'with cache {db_name}'
         else:
             cache_msg = f'with no cache'
-        if scicrunch_api is not None:
+        if npo:
+            self.__scicrunch = NpoSparql()
+            built = f" built at {self.__scicrunch.sckan_build()['release']}"
+            scicrunch_msg = f"using NPO{built}"
+        elif scicrunch_api is not None:
```

Comment:
Why are NPO and SciCrunch now mutually exclusive? My understanding is that we want to (optionally) get neuron connectivity from NPO instead of SciCrunch but to continue to use SciCrunch for everything else.

What happens in the NPO case when making a flatmap after deleting `knowledgebase.db` from the output directory? This will show if NPO (and its `nposparql.py` interface) has sufficient knowledge to replace the SciCrunch interfaces.

Reply:
I got it. Now NPO and SciCrunch are not mutually exclusive.